cms/static/sass/elements/_system-feedback.scss

```diff
@@ -554,15 +562,35 @@
     }
 
     .action-primary {
-      @include blue-button();
+      @extend %btn-primary-blue;
+      @extend %sizing;
 
-      @extend %t-strong;
+      .action-button-text {
+        display: inline-block;
+        vertical-align: baseline;
+      }
 
-      border-color: $blue-d2;
+      .icon {
+        display: inline-block;
+        vertical-align: baseline;
+      }
+
+      // CASE: new/create button
+      &.new-button,
+      &.button-new {
+        @extend %btn-primary-green;
+        @extend %sizing;
+      }
     }
 
     .action-secondary {
       @extend %t-action4;
+      color: $white;
```

Comment:
This change has also affected the "Cancel" button on the "Advanced Settings" page in Studio, and perhaps others. But I think it's better now - more readable? (FYI @cassiezamparini @jmakowski1123 in case you have feedback)

Before:

![Screenshot 2023-09-13 at 11 44 34 AM](https://github.com/openedx/edx-platform/assets/945577/c26e5188-fb35-4e08-9e9c-948fb8032b7b)

After:

![Screenshot 2023-09-13 at 11 45 18 AM](https://github.com/openedx/edx-platform/assets/945577/cf5b559c-74c8-4af5-9aa0-d253a45d21a8)

Reply:
I agree, I think it's more readable.